src/contracts/atlas/GasAccounting.sol

```diff
@@ -50,64 +59,53 @@ abstract contract GasAccounting is SafetyLocks {
     /// @notice Calculates the current shortfall between deficit (claims + withdrawals) and deposits.
```

Comment:
*Calculates the current shortfall between total debits (deposits + writeoffs) and credits (claims + withdrawals) to atlETH

Reply:
Updated it for latest changes